### ISSUE TITLE
[9.1.1] A previous solution appears in a new ticket

### ISSUE THREAD

**Issue by pcabreralallave** [glpi-project/glpi#1668]
Hello, we are having an issue with recently updated GLPI 9.1.1

A ticket is created in the normal way which is registered with "new" status.

When the technician enters the ticket to register a solution, he finds that there is already a previous solution entered, apparently is the solution of the previous ticket.

![image](https://cloud.githubusercontent.com/assets/19557972/22118373/7ad72ec8-de45-11e6-91a9-1f0aafdb72da.png)

As you can see the ticket is in "pending" status
![image](https://cloud.githubusercontent.com/assets/19557972/22118306/35bb4978-de45-11e6-9762-2b1f8228987f.png)

In history says that a solution has never been registered:

![image](https://cloud.githubusercontent.com/assets/19557972/22118316/41021820-de45-11e6-84aa-7bfe4ae7d5a8.png)

When you register a solution appears the solution of the previous ticket:

![image](https://cloud.githubusercontent.com/assets/19557972/22118414/a94c2bd2-de45-11e6-9a8e-9c15e6fa46cd.png)

**Comment by trasher** [comment]
I'm not able to reproduce on current 9.1/bugfixes branch. Maybe the issue has been solved already; or I miss informations on how to reproduce the issue.

**Comment by iamnoterp** [comment]
Hi there,
I'm able to reproduce this bug with 9.1.6 by doing the following:
1. Resolve existing ticket
2. Immediately create new ticket

![glpi - tickets - 276 - google chrome_319](https://user-images.githubusercontent.com/20347950/29008233-a3b7a01c-7b56-11e7-8436-94c13baadd3a.png)

However I've noticed that @pcabreralallave has the "More ticket" plugin; I disabled it in my install and can no longer recreate the issue.
